common/workunit/workunit.cpp

```diff
@@ -1042,6 +1043,7 @@ class CDaliWorkUnit : public CPersistedWorkUnit
         VStringBuffer path("/GraphProgress/%s", queryWuid());
         Owned<IRemoteConnection> conn = querySDS().connect(path, myProcessSession(), RTM_LOCK_WRITE|RTM_CREATE_QUERY, SDS_LOCK_TIMEOUT);
         IPTree *progress = ensurePTree(conn->queryRoot(), graphName);
+        // Actually, maybe it does work, but by luck?
```

Comment:
I don't think there's any real harm other than confusing me! Whether it's worth changing, I don't know.